docs/src/pages/docs/functions/helpers.md

```diff
@@ -293,7 +293,7 @@ functions, `dissoc` will remove any `undefined` values from the result.
 
 #### fanout
 
-`crocks/helpers/fanout`
+`crocks/Pair/fanout`
```

Comment:
This documentation should be moved to the `Pair` docs, and as part of that move will now require an example

Reply:
That also means that `fanout` will need to be removed from the tags list above, and added to the tags list in `Pair` (check out `branch` for an example.)